src/internal/middleware/logging/interceptor.go

```diff
@@ -98,6 +98,7 @@ func getCommonLogger(ctx context.Context, service, method string) context.Contex
 		}
 		if command := md.Get("command"); command != nil {
 			f = append(f, zap.Strings("command", command))
+			log.Info(ctx, "audit log: ", f...)
```

Comment:
That makes sense.  I'd omit the trailing ": ".  That marshals as something like `{"message":"audit log: "}` which looks kind of weird.

I also think that the user and request id are interesting items to add here; the command on its own doesn't seem that useful unless it's related somehow to the rest of the data stream.  (Remember that a busy pachd instance can be executing hundreds of RPCs in parallel, so the position of the log line won't necessarily make sense under those circumstances.)

Reply:
will remove the colon. user and request id should be captured in f already.

Reply:
Oh of course!